editor/editor_run.cpp

```diff
@@ -196,7 +197,7 @@ Error EditorRun::run(const String &p_scene, const String &p_custom_args, const L
 	for (List<String>::Element *E = args.front(); E; E = E->next()) {
 		printf(" %ls", E->get().c_str());
 	};
-	printf("\n");
+	putchar('\n');
```

Comment:
I'm not convinced that we gain anything by doing this.
We don't use `putchar()` in any Godot code so far, and this is not a core C library where we need to micro-optimize all operations, it's just a debug output when running a command (happens only once).